braga/README.md

```diff
@@ -0,0 +1,655 @@
+    + **Multi-media recording/editing studio** for video/podcast production
+    + "**Art Room**" for design, crafts & other arts.
+  + 1 Small ("single") room (_was previously used as a study_)
+    we intend to use this as our Laundry, utilities and linen room.
+  + Bathroom w/ Shower (_big enough for an accessible/wheelchair facility
+    or multiple showers_)
++ **First Floor**
+  + Extra-Large Living room (9.4 x 4.3m) - to be used as our main co-working
+  space. It can easily fit desks for 20 people (1m wide - see below)
+  + Dining Room fits 8 People
+  + Kitchen (large enough for double appliances and "American" fridges)
+  + Walk-in Pantry perfect for storing food and kitchen appliances.
+  + Office/Study - this will be our library and resource room (_printing etc_)
+  + Bathroom
++ **Second Floor**
+  + 4 Large bedrooms - the plan is to use these 3 large rooms as "Dorm" rooms
+    with bunk beds. Each room can comfortably fit 5 bunk beds with
+    storage space, however we are going to _initially_ limit each room to 4
+    (_double_) bunks (_see below for details_)
+  + 1 Medium bedroom this will be the bedroom for Ines & Nelson
+  or who ever is the "_house master_" once it's set up
+  and ready to run _independently_ of us.
+  + Bathroom w/ shower - this will be converted into 2 individual showers.
+  + Bathroom w/ bath - this will be converted into additional toilet cubicles.
++ **Outdoors / Garden**
+  + 3.5 - 4.5m around the house (_currently the garden is very neglected
+    so requires some attention/landscaping_) we are considering the following:
+    + Outdoor table Tennis (_covered between use_)
+    + Outdoor Table Football ("_fussball_" _also covered between use_)
+    + Hot tub (_we need to gauge demand for it
+      and cost it including annual maintenance_)
+  + Bird House (_disused_) - the plan is to convert it into something _useful_
+  like a sauna/steam room (_which is great for detox/letting off steam..._)
+  + Glass Cube for outdoor meditation, quiet reading and isolated creative work.
+
+## 1. The House for _Creative Technologists_
+
+![creative](https://cloud.githubusercontent.com/assets/194400/25162731/eb808bb6-24bc-11e7-9346-9cdbe128d3f7.png)
+
+
+`Braga House` is a place for people to live, learn and work
+with _Creative Technology_.
+
+A _haven_ for people who _already_ consider themselves a "***Digital Nomad***"
+and/or those who _want_ to _learn_ how to be ***Creative Technologists***.
+
+Initially the house will be "_invite-only_"
+(_because we want to be **selective** to establish the culture_) <br />
+but once we are up-and-running
+we will have an ***application process*** for people joining.
+
+### How Many Beds?
+
+Let's start by making one thing clear: the `Braga House` is _not "small"_.
+
+There is space for 64 beds (32 Bunks) in the house if we were to _maximise_ it,
+but our purpose is to make it _comfortable_ for everyone
+so our plan is to `limit` it to **32 beds** (16 bunks);
```

Comment:
No, 14.5 upstairs. 2 bunks in the basement room. measured.
i.e. the wheelchair accessible room 
(_so we don't have to install an elevator to the top floor..._)